packages/eslint-config-custom/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "eslint-config-custom",
+  "version": "0.0.0",
+  "main": "index.js",
+  "private": true,
+  "devDependencies": {
+    "eslint": "8.37.0",
+    "eslint-config-next": "13.3.0",
+    "eslint-config-prettier": "8.8.0",
```

Comment:
`gts` already brings `eslint` and `eslint-config-prettier` - do we need to install these here?